src/features/home/api/useGetOffersAndVenuesData.ts

```diff
@@ -0,0 +1,101 @@
+import { uniqBy } from 'lodash'
+import { useCallback, useEffect } from 'react'
+import { useQueries } from 'react-query'
+
+import { useAuthContext } from 'features/auth/context/AuthContext'
+import { useHomePosition } from 'features/home/helpers/useHomePosition'
+import { HomepageModule, isOffersModule, isVenuesModule } from 'features/home/types'
+import { useIsUserUnderage } from 'features/profile/helpers/useIsUserUnderage'
+import { SearchState } from 'features/search/types'
+import { VenueHit } from 'libs/algolia'
+import { fetchMultipleOffers } from 'libs/algolia/fetchAlgolia/fetchMultipleOffers/fetchMultipleOffers'
+import { useAdaptOffersPlaylistParameters } from 'libs/algolia/fetchAlgolia/fetchMultipleOffers/helpers/useAdaptOffersPlaylistParameters'
+import { fetchMultipleVenues } from 'libs/algolia/fetchAlgolia/fetchMultipleVenues'
+import { useTransformOfferHits, filterOfferHit } from 'libs/algolia/fetchAlgolia/transformOfferHit'
+import { useNetInfoContext } from 'libs/network/NetInfoWrapper'
+import { QueryKeys } from 'libs/queryKeys'
+import { Offer } from 'shared/offer/types'
+
+const isSearchState = (parameter: unknown): parameter is SearchState =>
+  typeof parameter === 'object' && parameter !== null
+
+export const useGetOffersAndVenuesData = (modules: HomepageModule[]) => {
+  const { position } = useHomePosition()
+  const transformHits = useTransformOfferHits()
+
+  const adaptedPlaylistParameters = useAdaptOffersPlaylistParameters()
```

Comment:
```suggestion
  const adaptPlaylistParameters = useAdaptOffersPlaylistParameters()
```
Car c'est la méthode qui va permettre d'adapter les paramètres des playlists, ce n'est pas ici qu'on les récupère, mais plutôt en ligne 33 où le naming `adaptedPlaylistParameters` serait plus approprié à la place de `parsedParameters`